docs/faq.md

```diff
@@ -11,32 +11,30 @@ For value transactions, there is the wallet.rs library which provides a stateful
 
 Additionally, there is also a native C ([iota.c](https://github.com/iotaledger/iota.c)) and an alternative, native javascript ([iota.js](https://github.com/iotaledger/iota.js/tree/chrysalis)) client library. 
 
-## What is Dust protection and how does it work?
+## What is Dust Protection and How Does it Work?
 
-Since IOTA is feeless and has the ability to send microtransactions, attackers could use this to spam the network with very low value transactions, which we call dust. To avoid this we only allow microtransaction below 1Mi of IOTA tokens to another address if you already have at least 1Mi on that address.
+Since IOTA is feeless and has the ability to send microtransactions, attackers could use this to spam the network with very low value transactions, which we call dust. To avoid this, we only allow microtransaction below 1Mi of IOTA tokens to another address if you already have at least 1Mi on that address.
 
 You can read more about Dust Protection [in the RFC here](https://github.com/iotaledger/protocol-rfcs/pull/32).
```

Comment:
- The provided link is for an open pull request, perhaps we should change this to a blog post or some documentation which explains this better.
- Instead of saying 'here', we could say 'this pull request' (or blog post/documentation if we change the link)

Reply:
So, we actually reference the RFCs a lot as its assumed the audience wants to go much around in our stuff. I agree that there _should_ be a reference to more digestible content but it doesnt exist at the moment :/